tests/integration/kubernetes/gha-run.sh

```diff
@@ -236,6 +239,76 @@ function cleanup() {
 	kubectl delete -f "${tools_dir}/packaging/kata-deploy/kata-rbac/base/kata-rbac.yaml"
 }
 
+function deploy_snapshotter() {
+	echo "::group::Deploying ${SNAPSHOTTER}"
+
+	case ${SNAPSHOTTER} in
+		nydus) deploy_nydus_snapshotter ;;
+		*) >&2 echo "${SNAPSHOTTER} flavour is not supported"; exit 2 ;;
+	esac
+
+	echo "::endgroup::"
+}
+
+function deploy_nydus_snapshotter() {
+    echo "deploy_nydus_snapshotter"
+	NODE_NAME=${NODE_NAME:-$(hostname | tr A-Z a-z)} 
+    kubectl config set-context --current --namespace=default
+	nydus_snapshotter_version=$(get_from_kata_deps "externals.nydus-snapshotter.version")
+    sed -i -e "s|ghcr.io/containerd/nydus-snapshotter:latest|ghcr.io/containerd/nydus-snapshotter:${nydus_snapshotter_version}|g" "${NYDUS_SNAPSHOTTER_DEPLOY_DIR}/nydus-snapshotter.yaml"
+
+    sed -i -e "s|CONTAINERD_CONFI_DIR|${CONTAINERD_CONFI_DIR}|g" "${tools_dir}/tools/nydus-snapshotter.yaml"
+    kubectl apply -f "${NYDUS_SNAPSHOTTER_DEPLOY_DIR}/nydus-snapshotter-rbac.yaml"
+    kubectl apply -f "${NYDUS_SNAPSHOTTER_DEPLOY_DIR}/nydus-snapshotter.yaml"
+	kubectl label node "$NODE_NAME" --overwrite nydus-snapshotter/nydus-snapshotter=true
```

Comment:
ca we add the label to worker nodes as https://github.com/confidential-containers/cloud-api-adaptor/blob/main/libvirt/install_operator.sh#L22
```
	local workers
	local label="nydus-snapshotter/nydus-snapshotter=true"

	workers="$(kubectl get nodes --no-headers | grep '\<worker\>' | awk '{ print $1 }')"
	for nodename in $workers; do
		if ! kubectl get "node/${nodename}" -o jsonpath='{.metadata.labels}' | \
			grep "$label" >/dev/null; then
			kubectl label node "$nodename" --overwrite "${label}"
		fi
	done
```

Reply:
Done.